From a8ea0edd9b6a95b07ea8a6d324f688cd727fd968 Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Fri, 14 Sep 2018 11:09:39 +0100
Subject: [PATCH] xfstests-dev: ensure xfstests_statx is defined

The definition of the inlined function xfstests_statx needs to be
moved to outside a #ifdef to ensure it always is defined.

Signed-off-by: Colin Ian King <colin.king@canonical.com>
---
 src/statx.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/statx.h b/src/statx.h
index 5d58208a..f6aaafd6 100644
--- a/src/statx.h
+++ b/src/statx.h
@@ -160,6 +160,8 @@ struct statx {
 
 #define STATX_ATTR_AUTOMOUNT		0x00001000 /* Dir: Automount trigger */
 
+#endif /* STATX_TYPE */
+
 static inline
 int xfstests_statx(int dfd, const char *filename, unsigned flags,
 		   unsigned int mask, struct statx *buffer)
@@ -172,5 +174,4 @@ int xfstests_statx(int dfd, const char *filename, unsigned flags,
 #endif
 }
 
-#endif /* STATX_TYPE */
 #endif /* STATX_H */
-- 
2.17.1

